thunder/benchmarks/benchmark_litgpt.py

```diff
@@ -393,7 +395,15 @@ def setup_compile(self, model):
 
                 executors.insert(0, transformer_engine_ex)
 
-            model = thunder.jit(model, executors=executors)
+            if "dynamo" in self.compile:
```

Comment:
wouldn't we want to tweak the config like
https://github.com/Lightning-AI/lightning-thunder/blob/a206e22605e09f2817c1c5b33ff1fbfb28ada1b7/thunder/benchmarks/benchmark_litgpt.py#L367-L370?

Reply:
I don't know why this cache size limit was added.

Reply:
hmm, in #940, torch.compile(fsdp2_model) seems to over the cache limit but the other configs look fine without the change.